### PR TITLE
Revert "[2.0.x] G33 clean up"

### PR DESCRIPTION
**Pull request by brandstaetter**
Reverts MarlinFirmware/Marlin#12648

This change unnecessarily removed 2 functions that are very useful for calibrating the Anycubic Kossel. With them gone, the user is reduced to guesswork. I scratched up my bed a bit trying to guess the offset...

### CODE REVIEW

**Comment by brandstaetter** [comment]
Pinging @LVD-AC - any objections?

**Comment by LVD-AC** [comment]
G33 P-1 was removed because the z_offset bug was fixed and this command is no longer required ALL the time. It did have a bug with deltas with a removable probe. So better do a G33 P1 and a paper test manualy and seperatly

G33 P1 is just removed from the LCD menu but still works (again with the offset bug fixed it is no longer required all the time)

G33 S was removed since the probing routine deals with that now. You can just use G33 in normal mode.

And the fix for the offset bug needs to remain in place (see comments).

**Comment by brandstaetter** [comment]
Which z_offset bug are you talking about exactly?

The bug with deltas and the removable probe (Anycubic Kossel) was fixed by adding an optional pause before deploy & stow so that the user could remove the probe before it is rammed into the bed.

G33 P-1 - I still think it's neat if you want to get the exact distance by that instead of having to guess and use babystepping...

G33 S - is the probing (delta calibration) routine now probing further down so that you don't need the height to be roughly correct? If so, then we can keep that as removed.

I still think getting the P-1 probe offset measurement back would be useful for me and other users. I am happy that the setup routine is simplified now (no extra height measurement).

**Comment by LVD-AC** [comment]
You needed to recalibrate the delta_height after each offset change, so G33 provided a way to change the offset and recalibrate the delta_height in one go. That is in the past now. Delta_height is now defined as the distance between home position and the z-min trigger point (so it remains constant). Z-offset adds (substacts) to that in order to go to the height of the 1st printed layer and is as such independent from delta_height. (printers without a probe do not have a z_offset since z_min trigger point is defined as the paper test height = 1st layer height).

This change made quite a few things obsolete or marginal at the best and the z-offset can be dealth with the same way as with cartesian printers . Babystepping is now fully functional and retains its value also after homing, etc...

Calibrate delta_height with the probe in place; remove the probe and babystep till the nozzle grabs a paper or during a print so it prints fine. This is exaclty what G33 P-1 was doing. But now the offset bug is fixed, the genuine babystepping routine works on deltas as it should, and there is no longer need for a special G33 alternative babystepping command to do that. (so it was removed)

In short: calibrate delta height (and all the other stuff) with the probe in place; and babystep (without the probe) till the nozzle is at the correct height and you're good to go. You just have to issue 2 commands (to do 2 completely different and unrelated things) instead of one combined (G33 P-1) : one for calibrating and one to set the z-offset.

**Comment by brandstaetter** [comment]
Understood, thanks. I will adapt the instructions. Thanks for the explanation!

**Comment by thinkyhead** [comment]
I'm adding the description above to the website, but I think I need to make it clearer with some visual aids. I'd like to put together a diagram showing where all the positions named in the description are:
- Delta Height
- Z Home Position
- Z-MIN trigger point
- Z Probe Offset
- "paper test height"
- "the height of the first printed layer"

The last two items may make more sense after I review the code of `G33` again. But I believe that "the height of the first printed layer" will vary according to how you slice your model. Unless this just refers to Z0.

**Comment by LVD-AC** [comment]
- first layer height = Z0.

I suggest making 2 diagrams : printers with a Zmin probe (and probe Zprobe offset) and without.

- for printers with a Zmin probe
delta height = ZHome position (max endstops trigger point) - Zmin trigger point
and
Z0 = Zhome postion - delta height - Z probe offset
Zmax = Zhome position = delta height - Zprobe offset

- for printers without 
delta height = Z Home position (max endstops trigger point) - paper test height.
and
Z0 = Z-home postion - delta height ( - 0)
Zmax = Zhome position = delta height (- 0)

If you want me to review the diagram...
